### PR TITLE
New LHC Apocalypse Checker Instant Answer

### PR DESCRIPTION
**Pull request by AlterationBrick**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [x] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

Adds an easter egg that determines whether the Large Hadron Collider has destroyed the world yet. Just for giggles; reject it if you don't like it, because it only took about an hour of my time.
###### Related Issues or Discussions

N/A
###### People to notify

@mention any relevant people/organizations

---

Instant Answer Page: https://duck.co/ia/view/has_the_large_hadron_collider_destroyed_the_world_yet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by mintsoft** [comment]
@AlterationBrick This LGTM, is there any work remaining?

**Comment by AlterationBrick** [comment]
@mintsoft All tests pass, but I was unable to test it on DuckPAN due to a "could not retrieve metadata" error. I might look into it in a few days when I have the time.

**Comment by mintsoft** [comment]
@AlterationBrick This should work in duckpan now, we've made metadata optional

**Comment by mintsoft** [comment]
I've tested this locally, it looks tip-top

**Comment by mintsoft** [comment]
Many Thanks @AlterationBrick Apologies this took so long to get around to

File: lib/DDG/Goodie/HasLHCDestroyedWorld.pm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+# ABSTRACT: Has the Large Hadron Collider destroyed the world yet? Nope.
+
+use DDG::Goodie;
+use strict;
+
+zci answer_type => 'has_lhcdestroyed_world';
+zci is_cached => 1;
+
+triggers any => 'has the large hadron collider destroyed the world',
```
Comment (moollaza) [inline lib/DDG/Goodie/HasLHCDestroyedWorld.pm]: @AlterationBrick @mintsoft this should be `startend`.

We should also ensure the remainder of the query is empty in the `handle` for good measure.

Comment (mintsoft) [reply lib/DDG/Goodie/HasLHCDestroyedWorld.pm]: Good point, I'll fix that in a min

Comment (mintsoft) [reply lib/DDG/Goodie/HasLHCDestroyedWorld.pm]: Fix: https://github.com/duckduckgo/zeroclickinfo-goodies/pull/4462

**Pull request merged**